### DISCUSSION TITLE
v3.9.0

### DISCUSSION THREAD

**Discussion by sojan-official** [chatwoot/chatwoot#9479] [Releases & Announcements]
## Changelog

- Performance improvements for reports (7x better load times on larger accounts)
- Improved performance for conversation filters
- Ability to export contacts with filters applied
- Ability to snooze conversations in bulk actions
- Support file uploads in the telegram channel
- Support for loom videos in help centre articles
- Numerous bug fixes and enhancements

###
Thanks to @JeffOtano, @MJRT, @clairton, @Maklu, @legioner0, @jderecho, @imtoanle  for the contributions

------

Notes: 

- This release involves critical db migration and adding indexes; please ensure migrations are run correctly
- Going forward, we will be deprecating `360Dialog` as a WhatsApp provider. Hence, new inboxes cannot be created with this provider starting from this release. Existing inboxes will continue to work, and a migration path will be provided
- This release contains billing-related changes that might reset unauthorised customisations of Chatwoot Enterprise editions.  

<hr /><em>This discussion was created from the release <a href='https://github.com/chatwoot/chatwoot/releases/tag/v3.9.0'>v3.9.0</a>.</em>

**Comment by tobsowo** [comment]
Does this release fix the auto deleting IG messages?

**Reply by sojan-official** [reply]
if you are referring https://github.com/chatwoot/chatwoot/pull/9287, This is merged

**Reply by tobsowo** [reply]
Alright

**Reply by tobsowo** [reply]
The IG message delete is not working on my end.

**Reply by evelyn-vnx** [reply]
Hi @sojan-official

I am trying to pass Meta's review for permissions approval.
When I delete a text message on Instagram, Chatwoot immediately shows "This message was deleted" replacing the deleted message. But in the case of images, audio or videos it shows the message "This message was deleted" but does not immediately remove them from the conversation in Chatwoot, I have to refresh the Chatwoot page to just stop the image, video from appearing.
![ig-delete](https://github.com/user-attachments/assets/29ba7f4d-41ef-42b1-b903-c7237709404d)

In the case of audio or video it is the same but when I refresh the Chatwoot page I still get a blank box in case of the deleted video and in the deleted audio as a 0 audio clip.
![delete-audio-video](https://github.com/user-attachments/assets/474c31dd-2837-4c28-86f2-5592022b53ea)

I am using Chatwoot version 3.9.0.

Has this already been corrected?

**Comment by Double-Click-ERP** [comment]
Hello @sojan-official ,

we are using "360Dialog" , and working now with v.3.9.0
what to do now ?

**Reply by Double-Click-ERP** [reply]
@sojan-official  ?

**Reply by vishnu-narayanan** [reply]
>Going forward, we will be deprecating 360Dialog as a WhatsApp provider. Hence, new inboxes cannot be created with this provider starting from this release. Existing inboxes will continue to work, and a migration path will be provided

ref: https://github.com/chatwoot/chatwoot/releases/tag/v3.9.0

**Reply by Double-Click-ERP** [reply]
how to do migration ?

**Reply by Double-Click-ERP** [reply]
@sojan-official 
@vishnu-narayanan 

how to do migration ?

**Reply by sojan-official** [reply]
the existing inboxes will continue to work. WE will provide a migration option for them down the road.   you won't be able to add new inboxes, thats all.

**Comment by glauberssoutodossantos** [comment]
Hi @sojan-official .

Next release it will be a major release? We will have chatwoot 4.0.0 next few months?

**Reply by sojan-official** [reply]
We don't have an ETA For 4.0.0 yet

**Comment by vigyhi5** [comment]
In the latest version of 3.9.0 the Bot Configuration from Inbox setting is completely missing, any specific reason?

**Comment by RCorsino** [comment]
I'm experiencing a timeout issue where Chatwoot calls my server for a webhook (on conversation create, on message send, etc.,). This causes to not send the message and keeps on failing. Is there a breaking change related to this? The webhook URL is configured on a per-inbox level rather than in the Integrations settings.

Didn't have a problem with this in the previous versions.
